### PR TITLE
Project library solo

### PR DESCRIPTION
**Pull request by kathinka**
book library

## Netlify link
[Netlify link
](https://projectlibrary.netlify.app/)

## Collaborators
flying solo

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,406 +1,501 @@
...
+	document.getElementById("user-input").value = "";
+	searchstring = userSearch;
+	if (searchstring != "") {
+		search(userSearch);
+	}
+}
+
+//get unique rating from the books array
+//get unique genres from the book array
+const genres = [...new Set(books.map((book) => book.genre))];
```
Comment (vittoriamatteoli) [inline script.js:235]: Nice!

File: script.js [hunk 2]
```diff
@@ -1,406 +1,501 @@
...
+const rating = [...new Set(books.map((book) => Math.trunc(book.rating)))];
```
Comment (vittoriamatteoli) [inline script.js:236]: Really clever!

File: script.js [hunk 3]
```diff
@@ -1,406 +1,501 @@
...
+
+//function for sort high to low rating
+function sortHigh(a, b) {
+	main.innerHTML = ""; //clear existing content;
+	const booksByHighRating = books.sort((a, b) => b.rating - a.rating);
+	displayFilteredBooks(booksByHighRating);
+}
+
+/*
+function getRandom() {
```
Comment (vittoriamatteoli) [inline script.js]: maybe you can take this out

**Review by vittoriamatteoli — COMMENTED** [review]
The code looks clean and follows all requirements. For an extra exercise, you could try to make the filtering work with the sorting simultaneously and add a"view all books " button to go back to the main homepage. Impressed!Nice work!

File: script.js (hunk 2 above)
Comment (kathinka) [reply script.js:236]: thank you :)

File: script.js (hunk 1 above)
Comment (kathinka) [reply script.js:235]: thank you :)

File: script.js (hunk 3 above)
Comment (kathinka) [reply script.js:331]: take what out? the function?

Comment (kathinka) [reply script.js]: ah!  yes - I forgot to remove it - thank you :)

File: .vscode/launch.json
Comment (NellaMori) [inline .vscode/launch.json:1]: this vscode folder, unless you want to share the settings with others, you can safely add it to your .gitignore file for example 👍

File: .vscode/settings.json
Comment (NellaMori) [inline .vscode/settings.json:1]: same here 🧹

File: index.html
Comment (NellaMori) [inline index.html:1]: Please use a formatter to have the correct spacing and indentation, and change the tab size settings from 4 to 2😄 you can see now there is a lot of empty spaces in the code that compromise the readability

File: script.js
Comment (NellaMori) [inline script.js:1]: same here, change indentation and format please 🪄

**Review by NellaMori — APPROVED** [review]
Hi Kathinka, well done finishing your Library app 📚 You followed the instructions and  the app fulfils the requirements of filtering and responsiveness. You also completed the stretch goals and even followed the design to inspire the UI of your app 🔥 
I left some comments in the code to improve the code structure by formatting it and removing unnecessary files, overall you did a good job 🚀

**Pull request closed without merging**